nautobot/dcim/templates/dcim/powerfeed_retrieve.html

```diff
@@ -1,168 +1,9 @@
-                <tr>
-                    <td>Voltage</td>
-                    <td>{{ object.voltage }}V</td>
-                </tr>
-                <tr>
-                    <td>Amperage</td>
-                    <td>{{ object.amperage }}A</td>
-                </tr>
-                <tr>
-                    <td>Phase</td>
-                    <td>{{ object.get_phase_display }}</td>
-                </tr>
-                <tr>
-                    <td>Max Utilization</td>
-                    <td>{{ object.max_utilization }}%</td>
-                </tr>
-            </table>
-        </div>
-{% endblock content_left_page %}
-
-{% block content_right_page %}
-        <div class="panel panel-default">
-            <div class="panel-heading">
-                <strong>Connection</strong>
-            </div>
-            {% if object.cable %}
-                <table class="table table-hover panel-body attr-table">
-                    <tr>
-                        <td>Cable</td>
-                        <td>
-                            {{ object.cable|hyperlinked_object }}
-                            <a href="{% url 'dcim:powerfeed_trace' pk=object.pk %}" class="btn btn-primary btn-xs" title="Trace">
-                                <i class="mdi mdi-transit-connection-variant" aria-hidden="true"></i>
-                            </a>
-                        </td>
-                    </tr>
-                    {% if object.connected_endpoint %}
-                        <tr>
-                            {% if object.connected_endpoint.device %}
-                                <td>Device</td>
-                                <td>{{ object.connected_endpoint.device|hyperlinked_object }}</td>
-                            {% else %}
-                                <td>Module</td>
-                                <td>{{ object.connected_endpoint.module|hyperlinked_object }}</td>
-                            {% endif %}
-                        </tr>
-                        <tr>
-                            <td>Power Port</td>
-                            <td>{{ object.connected_endpoint|hyperlinked_object }}</td>
-                        </tr>
-                        <tr>
-                            <td>Type</td>
-                            <td>{{ object.connected_endpoint.get_type_display|placeholder }}</td>
-                        </tr>
-                        <tr>
-                            <td>Description</td>
-                            <td>{{ object.connected_endpoint.description|placeholder }}</td>
-                        </tr>
-                        <tr>
-                            <td>Path Status</td>
-                            <td>
-                                {% if object.path.is_active %}
-                                    <span class="label label-success">Reachable</span>
-                                {% else %}
-                                    <span class="label label-danger">Not Reachable</span>
```

Comment:
Looking at the "After" screenshot, it looks like we're no longer formatting the path status as a label? Can we restore that?